tools/analysis/android_gfx

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+#    Copyright 2017 ARM Limited, Google and contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import pandas as pd
+from time import sleep
+from devlib.exception import TargetError
+from analysis_tool import AnalysisTool
+
+
+class GfxAnalysisTool(AnalysisTool):
```

Comment:
I would suggest to use the same name for both the class and the file. The name should also be "kind of" namespace based and details. Thus, I would say that `class AndroidGFX` should be better.

Moreover, I guess this is just an initial simple example and that you wanna add much more GFX related results to be generated by running this script, isn't it?

In that case I would suggest to feed-in some more example to better verify if the API works for even just slightly more complex use-cases.